samples/react-native-app/package.json

```diff
@@ -8,26 +8,30 @@
     "test": "jest"
   },
   "dependencies": {
-    "expo": "^45.0.5",
-    "expo-splash-screen": "^0.2.3",
-    "expo-updates": "^0.13.2",
-    "react": "~16.9.0",
-    "react-dom": "~16.9.0",
-    "react-native": "^0.68.2",
-    "react-native-gesture-handler": "~1.6.0",
-    "react-native-reanimated": "~2.10.0",
-    "react-native-screens": "~2.2.0",
+    "expo": "^49.0.16",
+    "expo-splash-screen": "^0.20.0",
+    "expo-updates": "~0.18.16",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.72.6",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-reanimated": "~3.3.0",
+    "react-native-screens": "~3.22.0",
     "react-native-unimodules": "~0.9.0",
-    "react-native-web": "^0.17.7"
+    "react-native-web": "~0.19.6",
+    "@expo/webpack-config": "^19.0.0"
   },
   "devDependencies": {
-    "@babel/core": "~7.9.0",
+    "@babel/core": "~7.23.2",
     "babel-jest": "~25.2.6",
-    "jest": "^28.1.1",
+    "jest": "^29.2.1",
     "react-test-renderer": "~16.9.0"
   },
   "jest": {
     "preset": "react-native"
   },
+  "overrides": {
```

Comment:
this is the same overrides as what's in the main package, helps us get around the fact that lots of packages that depend on semver haven't been updated.